File: app/src/main/java/com/annapol04/munchkin/engine/Monsterlevel.java
package com.annapol04.munchkin.engine;

import android.support.annotation.DrawableRes;
import android.support.annotation.StringRes;

/**
 * Created by Admin on 09.03.2018.
 */

public class Monsterlevel extends Card {

    public int monsterBonus;

    public Monsterlevel (@StringRes int name, @StringRes int description, @DrawableRes int imageResourceID, int bonus) {
        super(name, description, imageResourceID);
        this.monsterBonus = bonus;
    }
}

File: app/src/main/java/com/annapol04/munchkin/engine/Monsterlevel.kt
package com.annapol04.munchkin.engine

import android.support.annotation.DrawableRes
import android.support.annotation.StringRes

class Monsterlevel(@StringRes name: Int, @StringRes description: Int, @DrawableRes imageResourceID: Int, var monsterBonus: Int)
    : Card(name, description, imageResourceID)
